go.mod

```diff
@@ -10,12 +10,12 @@ require (
 	github.com/prometheus/client_golang v1.5.1
 	github.com/sirupsen/logrus v1.4.2
 	github.com/stretchr/testify v1.4.0
-	k8s.io/api v0.16.9
-	k8s.io/apimachinery v0.16.9
-	k8s.io/client-go v0.16.9
-	k8s.io/code-generator v0.16.9
+	k8s.io/api v0.16.15
+	k8s.io/apimachinery v0.16.15
+	k8s.io/client-go v0.16.15
+	k8s.io/code-generator v0.18.3
```

Comment:
code-generator version should match with other k8s versions

Reply:
this is a minor issue.

Reply:
> this is a minor issue.

Oh, yes. Sorry for ignoring the mismatch. It seems that it is modified automatically whe executing `go mod vendor`. I'll submit another pr for it.

Reply:
never mind. probably due to some dependency issue. We can clean up dependencies together